pkgs/fc/ceph/src/fc/ceph/main.py

```diff
@@ -334,6 +346,7 @@ def main(args=sys.argv[1:]):
 
     # extract parsed arguments from object into a dict
     args = vars(parser.parse_args(args))
+    print(args)
```

Comment:
This looks like debugging code?

Reply:
Indeed, I somehow missed to commit the removal.